Magento2/ruleset.xml

```diff
@@ -522,6 +522,31 @@
     <rule ref="Squiz.WhiteSpace.SuperfluousWhitespace.EmptyLines">
         <severity>0</severity>
     </rule>
+    <rule ref="Magento2.GraphQL.ValidTypeName">
+        <severity>6</severity>
+        <type>warning</type>
+    </rule>
+    <rule ref="Magento2.GraphQL.ValidFieldName">
+        <!--
+         * Field name validation deactivated as per request
+         * https://github.com/magento/magento-coding-standard/pull/141#discussion_r323349939
+        -->
+        <exclude name="Magento2.GraphQL.ValidFieldName" />
+        <severity>6</severity>
```

Comment:
If this rule needs to be deactivated please set the severity to 0.

Reply:
Fixed in https://github.com/magento/magento-coding-standard/pull/141/commits/c539ed44c3b93284d9bc24e5e58c0271cbaa1d40